_posts/2017-07-19-runners-driver.md

```diff
@@ -0,0 +1,52 @@
+---
+title: "Concordion | Runners and Drivers"
+heading: "Runners and Drivers"
+description: "Layering test code with Runners and Drivers"
+identifier: "runners-drivers"
+author: "Nigel Charman"
+homepage: http://tutansblog.blogspot.co.nz/
+---
+
+Designing well structured test code can lead to effective reuse of the application drivers across test suites, as well as open up their use for exploratory and maintenance scripting. It avoids lock-in to any specific test runner or driver, opening up the potential for migrating to new test runners or drivers.
```

Comment:
Please change this paragraph to:

This article describes the Runners and Drivers pattern which enables test code to be layered with clean interfaces. This enhances maintainability of the code and leads to effective reuse of the application drivers across test suites. In addition, it opens up the drivers for use in exploratory and maintenance scripting. It also avoids lock-in to any specific test runner or driver, opening up the potential for future migration to new test runners or drivers.